docs/servers.rst

```diff
@@ -141,7 +150,7 @@ Ubuntu installer.
 If your USB is mapped to /dev/sdX and you are currently in the directory that
 contains the Ubuntu ISO, you would use dd like so: ::
 
-   sudo dd conv=fdatasync if=ubuntu-16.04.7-server-amd64.iso of=/dev/sdX
+   sudo dd conv=fdatasync if=ubuntu-20.04.2-live-server-amd64.iso of=/dev/sdX
```

Comment:
It may be worth explicitly emphasizing that one must specify the device, not a partition like `/dev/sdb1`. Yes, I'm saying this because I was just bitten by it. :)